gprofiler/utils.py

```diff
@@ -242,6 +243,7 @@ def get_libc_version() -> Tuple[str, bytes]:
 
 def log_system_info():
     uname = platform.uname()
+    logger.info(f"gProfiler Python version: {sys.version}")
```

Comment:
This will return that `gprofiler` itself run on.
It will be more useful return python version of process which being profiled.

Reply:
Yeah it's on purpose - see "Motivation and context" :)

I also plan to add logging of the profiled processes - this is easily done by passing the `RUST_LOG=info` environment variable, it makes py-spy write some logs which include the version. Later on we might make it more structures.

Reply:
Ohh ok, got it. :)